contracts/LinniaRecords.sol

```diff
@@ -9,28 +9,39 @@ import "./LinniaUsers.sol";
 contract LinniaRecords is Ownable {
     using SafeMath for uint;
 
-    struct FileRecord {
-        address fileOwner;
-        uint sigCount;
+    // Struct of a linnia record
+    // A linnia record is identified by its root hash, which is
+    // sha(sha(data), sha(metadata))
+    struct Record {
+        // owner of the record
+        address owner;
+        // hash of the data before its encrypted
+        bytes32 dataHash;
+        // hash of the plaintext metadata
+        bytes32 metaHash;
+        // attestator signatures
         mapping (address => bool) sigs;
+        // count of attestator sigs
+        uint sigCount;
+        // calculated iris score
```

Comment:
I think it makes sense for users to downvote a file, but maybe a separate field for negative iris score since `uint` is easier to work with.
Maybe in future versions